src/pages/public/Login/Login.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { useAuth } from '../../../context/auth/authContext';
+import { formSubmitHandler } from '../../../service';
+import './Login.css';
+
+export const Login = () => {
+  const { authDispatch } = useAuth();
+  const [user, setUser] = useState({ email: '', password: '' });
+  const navigate = useNavigate();
+  const loginHandler = async e => {
+    e.preventDefault();
+    try {
+      const response = await formSubmitHandler(
+        '/api/auth/login',
+        user,
+        authDispatch
+      );
+      if (response === 200) {
+        navigate('/', { replace: true });
+      } else if (response === 401) {
+        alert('Credentials are wrong');
+      } else if (response === 404) {
+        alert('You need to sign up, coz you are not registered');
+      } else {
```

Comment:
You don't need to check for multiple status, you can just throw an error and in catch(error) you can console.log all types of errors.

Reply:
But the catch block runs only when there is a network error, right?.